src/main/groovy/oe/espresso/latte/AblUnit.groovy

```diff
@@ -401,8 +401,11 @@ class AblUnit extends BaseLatteSourceTask {
                 if (!profilerOutputDir) {
                     profilerOutputDir = new File(project.buildDir, "profiler")
                 }
+                // make sure the directory exists
+                project.fileTree(profilerOutputDir)
```

Comment:
this doesn't create physical files on the disk.  this is a "getter" function that builds an in memory representation of the files that may already be on disk

https://docs.gradle.org/current/javadoc/org/gradle/api/file/FileTree.html

Reply:
yes. but "filetree" is "in memory" not "on disk".  this won't modify the file system.  since you need PCT to write a file to disk, the directory needs to exist,  if PCT is already creating the folder then you don't need to do this, but i've seen many cases where PCT doesn't actually create the folder (it is lazy).

Reply:
also filetree() function returns an object...but your code just throws away that object.

Reply:
I believe this is what you want:

https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#mkdir-java.lang.Object-

Reply:
Updated to use `File.mkdirs`